sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_management_link_async.py

```diff
@@ -228,17 +228,16 @@ async def execute_operation(
             message.application_properties["locales"] = locales
         try:
             # TODO: namedtuple is immutable, which may push us to re-think about the namedtuple approach for Message
-            new_properties = message.properties._replace(message_id=self.next_message_id)
+            new_properties = message.properties._replace(message_id=uuid4())
```

Comment:
the message id can be something random ? It doesnt need to corelate during completions or other checks

Reply:
no, Go is also generating a random ID